src/builtin.ts

```diff
@@ -0,0 +1,19 @@
+// calculates bitwise 'AND' of given two inputs x and y
+function BITWISE_AND(x: bigint, y: bigint): bigint {
+  return x & y;
+}
+
+// calculates bitwise 'OR' of given two inputs x and y
+function BITWISE_OR(x: bigint, y: bigint): bigint {
+  return x | y;
+}
+
+// calculates bitwise 'XOR' of given two inputs x and y
+function BITWISE_XOR(x: bigint, y: bigint): bigint {
+  return x ^ y;
+}
+
+// calculates bitwise 'AND', 'XOR' and 'OR' of given two inputs x and y
+function BITWISE_OPERATIONS(x: bigint, y: bigint): [bigint, bigint, bigint] {
+  return [BITWISE_AND(x, y), BITWISE_XOR(x, y), BITWISE_OR(x, y)];
+}
```

Comment:
this will not be useful in our context as the FUNCTION will directly be written in the cell